src/sentry/integrations/utils/code_mapping.py

```diff
@@ -114,7 +118,7 @@ def _find_code_mapping(self, frame_filename: FrameFilename) -> Union[CodeMapping
             return None
         # This means that the file has been found in more than one repo
         elif len(_code_mappings) > 1:
-            logger.warning(f"More than one file matched for {frame_filename.full_path}")
+            logger.warning(f"More than one repo matched {frame_filename.full_path}")
```

Comment:
I have added a new test that tests this code. The text was incorrect as the file shows up in more than a repo rather than showing more than once in the same repo.